src/Raven.Server/Documents/PeriodicBackup/Restore/RestoreBackupTask.cs

```diff
@@ -523,14 +523,26 @@ private List<string> GetFilesForRestore(string backupLocation)
                 },
                 onDatabaseRecordAction: smugglerDatabaseRecord =>
                 {
-                    // need to enable revisions before import
-                    database.DocumentsStorage.RevisionsStorage.InitializeFromDatabaseRecord(smugglerDatabaseRecord);
-
+                    databaseRecord.ConflictSolverConfig = smugglerDatabaseRecord.ConflictSolverConfig;
+                    foreach (var setting in smugglerDatabaseRecord.Settings)
+                    {
+                        databaseRecord.Settings[setting.Key] = setting.Value;
```

Comment:
Aren't you overwriting the `DataDir` here?

Reply:
No, I am not saving the DataDir in the backup